### PR TITLE
Feature/529: Guardians should be able to mark activities as complete like the citizens can

### LINKED ISSUE #529: As a guardian I would like to be able to mark activities as completed
**Is your feature request related to a problem? Please describe.**
Guardians should be able to mark activities as complete like the citizens can.

**Describe the solution you'd like**
To be added.

### PR DESCRIPTION
**Pull request by LJensen0403**
Rendered complete activity button if user is a guardian.
Widget tests if the button is rendered.

closes #529

### CODE REVIEW

**Comment by Determinatoro** [comment]
Please use the following checklist, @ZohraAmini. Feel free to check it off as you go along.

**Code Design**
- [x] The code is in the right place? (Both in terms of folder structure and class structure)
- [x] The code is not over-engineered. Examples of over-engineering: Implemented behavior for future problems
- [x] This code could not, to the best of my knowledge, have reused existing code
- [ ] The code does not introduce functionality that is necessary for solving the problem

**Code Readability and Maintainability**
- [x] Names are meaningful and self-documenting
- [x] It is understandable, by reading the code, what it does
- [x] The code is simple and readable. i.e. it contains no "hack", or obscure solution
- [x] All new functionality is covered by tests
- [x] It is understandable, by reading the tests, what they do
- [x] The tests covers sensible cases. Both happy paths and exception paths
- [x] The tests cover the full functionality. i.e., The tests fails if some of the requested functionality is missing
- [ ] Error messages in the code are understandable
- [ ] Confusing sections of code are covered by comments

**Functionality**

For this part, open the app, and test:

- [x] The features that the code claims to implement, are actually implemented
- [x] The code does not look like it has bugs
- [ ] All the new screens are reachable through in-app navigation

**Points for Discussion and Reflection**

- [ ] UI messages, that the user might meet, are actually helpful
- [ ] There are no errors that are bad practice. E.g. hard-coding a test database connection
- [ ] There is well written code in this. Remember to tell the developer!

**Comment by Determinatoro** [comment]
Please use the following checklist, @JeppeKLau. Feel free to check it off as you go along.

**Code Design**
- [x] The code is in the right place? (Both in terms of folder structure and class structure)
- [ ] The code is not over-engineered. Examples of over-engineering: Implemented behavior for future problems
- [x] This code could not, to the best of my knowledge, have reused existing code
- [ ] The code does not introduce functionality that is necessary for solving the problem

**Code Readability and Maintainability**
- [x] Names are meaningful and self-documenting
- [x] It is understandable, by reading the code, what it does
- [ ] The code is simple and readable. i.e. it contains no "hack", or obscure solution
- [x] All new functionality is covered by tests
- [x] It is understandable, by reading the tests, what they do
- [x] The tests covers sensible cases. Both happy paths and exception paths
- [x] The tests cover the full functionality. i.e., The tests fails if some of the requested functionality is missing
- [x] Error messages in the code are understandable
- [x] Confusing sections of code are covered by comments

**Functionality**

For this part, open the app, and test:

- [x] The features that the code claims to implement, are actually implemented
- [x] The code does not look like it has bugs
- [x] All the new screens are reachable through in-app navigation

**Points for Discussion and Reflection**

- [x] UI messages, that the user might meet, are actually helpful
- [x] There are no errors that are bad practice. E.g. hard-coding a test database connection
- [ ] There is well written code in this. Remember to tell the developer!

File: lib/screens/show_activity_screen.dart [hunk 1]
```diff
@@ -630,14 +630,17 @@ class ShowActivityScreen extends StatelessWidget {
                       return const CircularProgressIndicator();
                     }
                     if (weekplanModeSnapshot.data == WeekplanMode.guardian) {
-                      return GirafButton(
-                          key: const Key('CancelStateToggleButton'),
+                      return Container(
+                        child: Row(children: <Widget>[
+                        Container(child:
+                        GirafButton(
+                        key: const Key('CancelStateToggleButton'),
```
Comment (JeppeKLau) [inline lib/screens/show_activity_screen.dart]: This seems hacky and overly engineered. Why not just stick to the old button?

**Review by JeppeKLau — CHANGES_REQUESTED** [review]

File: lib/screens/show_activity_screen.dart (hunk 1 above)
Comment (LJensen0403) [reply lib/screens/show_activity_screen.dart]: Can you please elaborate on what you mean by "stick to the old button"? 
To start, both citizens and guardians only had a single button each. To be able to return two buttons for guardians, we had to wrap them in a container.
The nested containers are a byproduct of a padding issue, but we are working on solving it

**Review by JeppeKLau — APPROVED** [review]

**Review by ZohraAmini — APPROVED** [review]

**Pull request merged**